README.md

```diff
@@ -1,3 +1,7 @@
+## Task2
+S3: http://my-aws-fe.s3-website-eu-west-1.amazonaws.com/ (403 because cloud front setup)
```

Comment:
doesn't work the link 
I see  not found 404 
![image](https://user-images.githubusercontent.com/33509376/112854451-897c3b80-90b6-11eb-9d9a-d0ee02ea62b6.png)